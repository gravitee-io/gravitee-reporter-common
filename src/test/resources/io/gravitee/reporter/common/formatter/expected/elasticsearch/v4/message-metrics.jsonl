{ "index" : { "_index" : "gravitee-v4.metric.MessageMetrics-2023.08.28", "_id" : "10e949d4-fa38-4ff4-a139-1358bbacd127-entrypoint"} }
{ "gateway":"gateway-id" ,"@timestamp":"2023-08-23T13:12:01.153Z" ,"request-id":"1d1d007b-7891-45d0-9d00-7b789175d0a2" ,"api-id":"be926300-20aa-4109-9263-0020aa31096c" ,"client-identifier":"b6cf5bd6081fb775fc3f89927e582d7e945f955c35b73a07879d75bb4ea96a01" ,"correlation-id":"10e949d4-fa38-4ff4-a139-1358bbacd127" ,"operation":"subscribe" ,"connector-type":"entrypoint" ,"connector-id":"http-get" ,"content-length":4 ,"count":1 ,"gateway-latency-ms":0 ,"custom": { "zone":"europe-west10-a" } }
