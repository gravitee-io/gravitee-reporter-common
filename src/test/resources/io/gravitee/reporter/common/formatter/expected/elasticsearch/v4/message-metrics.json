{
  "gateway": "gateway-id",
  "_id": "10e949d4-fa38-4ff4-a139-1358bbacd127-entrypoint",
  "type": "v4-message-metrics",
  "date": "2023.08.23",
  "@timestamp": "2023-08-23T13:12:01.153Z",
  "request-id": "1d1d007b-7891-45d0-9d00-7b789175d0a2",
  "api-id": "be926300-20aa-4109-9263-0020aa31096c",
  "client-identifier": "b6cf5bd6081fb775fc3f89927e582d7e945f955c35b73a07879d75bb4ea96a01",
  "correlation-id": "10e949d4-fa38-4ff4-a139-1358bbacd127",
  "operation": "subscribe",
  "connector-type": "entrypoint",
  "connector-id": "http-get",
  "content-length": 4,
  "count": 21,
  "count-increment": 10,
  "gateway-latency-ms": 0,
  "custom": {
    "zone": "europe-west10-a"
  }
}
